workspace/user_test.go

```diff
@@ -24,14 +26,14 @@ func TestAdd(t *testing.T) {
 	api := houston.NewHoustonClient(client)
 	id := "ck1qg6whg001r08691y117hub"
 	role := "test-role"
-	email := "andrii@test.com"
+	email := "andrii@apple.com"
```

Comment:
@adam2k i'm not sure that we need this change.

Reply:
That test fails unless I change that because @test.com doesn't have any MX records.

Reply:
and it's the idea of test. i disagree that we want to test MX record.
cc @sjmiller609